apps/arena/lib/arena/matchmaking/deathmatch_mode.ex

```diff
@@ -0,0 +1,152 @@
+defmodule Arena.Matchmaking.DeathmatchMode do
+  @moduledoc false
+  alias Arena.Utils
+  alias Ecto.UUID
+
+  use GenServer
+
+  # 3 Mins
+  @match_duration 180_000
+  @respawn_time 5000
+
+  # Time to wait to start game with any amount of clients
+  @start_timeout_ms 4_000
```

Comment:
Create an issue and link it here to add these params to the Configurator.

Reply:
Done in #985